lib/sycamore/sycamore/connectors/writers/common.py

```diff
@@ -140,6 +143,6 @@ def _empty_dict_filter(x: Any) -> bool:
 
 def _make_type_filter(types: list[type]) -> Callable[[Any], bool]:
     def _type_filter(x):
-        return isinstance(x, tuple(types))
+        return not isinstance(x, tuple(types))
```

Comment:
Having an exclusion check is potentially more inefficient since it guarantees having to check all values, could be worth having both an inclusion and exclusion check if possible